.github/ISSUE_TEMPLATE/1-issue.yml

```diff
@@ -0,0 +1,66 @@
+name: "✅ Bug Report"
+description: File a bug report.
+title: "[Bug]: "
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## Thank You for taking the time to fill out this bug report! The more information you provide, the faster we can help.
+        
+        Please remember, a bug report is _not the place to ask questions_. You can
+        use [Discord](https://discord.gg/YH8Pqja2) for that, or start a topic in [GitHub
+        Discussions](https://github.com/wpengine/hwptoolkit/discussions).
+        
+        Also please note that _not all the fields are required_, but _the more information_ you provide, _the better_ we can help you.
+
+  - type: textarea
+    id: what-did
+    attributes:
+      label: What did you do?
+      description: Steps you took that led to the issue
+      placeholder: I used the `usePost` hook to fetch a post with ID 123 from my WordPress site
+    validations:
+      required: true
+
+  - type: textarea
+    id: what-expected
+    attributes:
+      label: What did you expect to happen?
+      description: The behavior you anticipated
+      placeholder: The hook should have returned the post data and set isLoading to false
+    validations:
+      required: true
+
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What actually happened?
+      description: The unexpected behavior you observed
+      placeholder: The hook returned undefined data but isLoading remained true indefinitely, even though the network request completed successfully
+    validations:
+      required: true
+
+  - type: textarea
+    id: additional-context
+    attributes:
+      label: Additional context (Optional)
+      description: Add any other context about the problem here, such as screenshots, error logs, etc.
```

Comment:
Let's add a request for a minimal reproduction here: https://stackoverflow.com/help/minimal-reproducible-example

Be sure to make that link available to folks

Reply:
Added separate textarea input specifically for the MRE